sdk/include/psdk/dwmapi.h

```diff
@@ -74,12 +104,35 @@ enum DWMFLIP3DWINDOWPOLICY {
     DWMFLIP3D_LAST
 };
 
+typedef enum DWMTRANSITION_OWNEDWINDOW_TARGET {
+    DWMTRANSITION_OWNEDWINDOW_NULL = -1,
+    DWMTRANSITION_OWNEDWINDOW_REPOSITION = 0
+};
+
 typedef enum _DWM_SOURCE_FRAME_SAMPLING {
     DWM_SOURCE_FRAME_SAMPLING_POINT,
     DWM_SOURCE_FRAME_SAMPLING_COVERAGE,
     DWM_SOURCE_FRAME_SAMPLING_LAST
 } DWM_SOURCE_FRAME_SAMPLING;
 
+typedef enum GESTURE_TYPE {
+    GT_PEN_RIGHTTAP,
+    GT_PEN_DOUBLETAP,
+    GT_PEN_RIGHTTAP,
+    GT_PEN_PRESSANDHOLD,
+    GT_PEN_PRESSANDHOLDABORT,
+    GT_TOUCH_TAP,
+    GT_TOUCH_DOUBLETAP,
+    GT_TOUCH_RIGHTTAP,
+    GT_TOUCH_PRESSANDHOLD,
+    GT_TOUCH_PRESSANDHOLDABORT,
+    GT_TOUCH_PRESSANDTAP
+};
+
+typedef enum WINDOWCOMPOSITIONATTRIB {
```

Comment:
This name needs to be different from the one used at the end of the `enum` definition.

```suggestion
typedef enum _WINDOWCOMPOSITIONATTRIB {
```

Reply:
got it. Will push a fix shortly